apps/fiona/compute_gain.cpp

```diff
@@ -991,7 +991,7 @@ int main(int argc, char const ** argv)
     bool error = false;
 
     // Format recognition tag used for sequence file I/O.
-    seqan::AutoSeqFormat seqFormatTag;
+    // seqan::AutoSeqFormat seqFormatTag; // deprecated
```

Comment:
Not that I can tell. I originally commented it out to see any warnings or error that might follow but nothing happend :no_mouth: ..